velox/functions/prestosql/window/tests/AggregatesWindowTest.cpp

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
Nit : Remove 's' in the naming. It is AggregateWindow not AggregatesWindow

Reply:
Apologies for missing the 's', updated.